CONTRIBUTING.md

```diff
@@ -232,42 +233,44 @@ See [Debugging Tips](README.md#debugging-tips) in the readme.
 
 # For Project Maintainers
 
-## Releasing to NPM
+## Releasing to npm
+
+Releasing to npm consists of the following phases:
 
-Releasing to NPM consists of 3 phases:
 1. Source Code: mark a release.
     1. Bump `package.json` version following the SEMVER rules and send a PR titled `'chore: mark version vXXX.YYY.ZZZ'` ([example](https://github.com/GoogleChrome/puppeteer/commit/808bf8e5582482a1d849ff22a51e52024810905c)).
```

Comment:
Drive-by question: are PRs like this created manually? I'm asking because the `<!-- GEN:empty-if-release -->`-style comments imply otherwise.

Reply:
Thanks! I'll submit a follow-up CL that explicitly calls out that one needs to run `npm run doc` after bumping the `package.json` version number.